lib_xua/src/core/endpoint0/xua_endpoint0.c

```diff
@@ -820,7 +820,9 @@ void XUA_Endpoint0_loop(XUD_Result_t result, USB_SetupPacket_t sp, chanend c_ep0
             case USB_BMREQ_H2D_CLASS_EP:
             case USB_BMREQ_D2H_CLASS_EP:
                 {
+#if (NUM_USB_CHAN_OUT > 0) || (NUM_USB_CHAN_IN > 0)
```

Comment:
Very minor but I see:
NUM_USB_CHAN_OUT > 0
and
(NUM_USB_CHAN_OUT != 0

Maybe settle on one. I think NUM_USB_CHAN_OUT > 0 reads better